plugins/meson.build

```diff
@@ -12,6 +12,7 @@ subdir('solokey')
 subdir('synaptics-cxaudio')
 subdir('synaptics-prometheus')
 subdir('test')
+subdir('tpm-eventlog')
```

Comment:
Don't forget to add a README.md to this directory.

Reply:
Added, thanks.